src/wizard/apple/index.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Apple (Cocoa)
```

Comment:
Not sure if Apple is the right name here, but I defer to others' expertise. 

```suggestion
name: iOS/OS (Cocoa)
```

Reply:
theres already an iOS page so im not super concerned about that. "OS" doesnt really make a ton of ssense here since we're only show apple/cocoa

Reply:
If listing actual platforms, consider we have:

iOS, watchOS, tvOS, iPadOS.. macOS

Reply:
I wonder if Apple announced a new one today

Reply:
they announced new date for these and i might have bought a watch

Reply:
LOL! Last time we talked in person, I bought one too.
Good buy